### PR TITLE
CNV-16642: Release note text for CNV dashboard

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with https://issues.redhat.com/browse/CNV-16642.
It applies to the CNV 4.10 release.

### CODE REVIEW

**Comment by lmandavi** [comment]
@sradco - Please approve by adding LGTM.
https://deploy-preview-42682--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-10-release-notes.html#virt-4-10-web-new

File: virt/virt-4-10-release-notes.adoc [hunk 1]
```diff
@@ -95,6 +95,9 @@ The SVVP Certification applies to:
 //[id="virt-4-10-changes"]
 //== Notable technical changes
 
+//CNV-16642 - Dashboard feature.
+* The {VirtProductName} dashboard provides data on resource consumption for virtual machines and associated pods. The visualization metrics displayed in the {VirtProductName} dashboard are based on xref:../virt/logging_events_monitoring/virt-prometheus-queries.adoc#virt-prometheus-queries[Prometheus Query Language (PromQL) queries].
```
Comment (ousleyp) [inline virt/virt-4-10-release-notes.adoc:99]: A couple of phrasing suggestions for you to consider:

s/provides data on resource consumption/displays resource consumption data

s/The visualization metrics displayed/The metrics that are represented

**Review by ousleyp — COMMENTED** [review]
Overall this LGTM, I just have a couple of wording suggestions. :) Please squash to one commit when you're ready for me to merge. Thanks!

File: virt/virt-4-10-release-notes.adoc (hunk 1 above)
Comment (lmandavi) [reply virt/virt-4-10-release-notes.adoc:99]: Plan to incorporate the first suggestion. I am not seeing much  value gained by changing the original phrasing for the second item, therefore retaining it.

**Pull request merged**